### PR TITLE
adds a fix for ubuntu18.04, also adds back ubuntu18.04 tests

### PR DESCRIPTION
**Pull request by jamesbeedy**
* appends the venv where the python yaml lives to the path
* re-adds tests for ubuntu18.04

### CODE REVIEW

**Comment by egee-irl** [comment]
Adding the `path.append` before the `from` and `imports` makes Flake8 angry. However, adding it _after_ the imports causes the whole thing to fail.

One option is to tell Flake to ignore the error but if we are telling our linter to ignore hacks and other issues, why run the linter at all?

**Comment by jamesbeedy** [comment]
I think ignoring E402 is entirely legitimate here until we figure out another way to add the python modules to the path for when the configure hook executes.

Skipping lint for imports not at the top of the file is not the end of the world here :)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# VARIABLES
+export PATH := /snap/bin:$(PATH)
+
+# TARGETS
+lint: ## Remove .tox and build dirs
+	@tox -e lint
+
+# Display target comments in 'make help'
+help: 
+	grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
```
Comment (egee-irl) [inline Makefile:10]: Is this makefile intended to be run outside of CI?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[tox]
```
Comment (egee-irl) [inline tox.ini:1]: What do we gain by adding Tox at this stage of development? We literally have a _single_ Python script that runs once during the initial configuration of the Snap

Comment (jamesbeedy) [reply tox.ini:1]: Fair enough. I think you make a good case to remove tox in favor of just calling out the commands.

**Pull request merged**